src/main/java/com/mraof/minestuck/computer/ComputerReference.java

```diff
@@ -9,7 +9,7 @@
 
 public interface ComputerReference
 {
-	static ComputerReference of(ComputerTileEntity te)
+	static ComputerReference of(ComputerBlockEntity te)
```

Comment:
on line 20 below, is it appropriate to change tile_entity to block_entity if I also change it in TEComputerReference? Also may TEComputerReference be renamed to BEComputerReference?

Reply:
Yes and yes

Reply:
And to double confirm, I do not need to change that string(tile_entity) in any other file to the best of your knowledge?

Reply:
Indeed, that nbt key should only be used in those two places.

Reply:
If thats the case then there are no other issues I can detect with this PR